### PR TITLE
World bubble map POC

### PR DESCRIPTION
**Pull request by janpipek**
Solving #13 ...

### CODE REVIEW

**Comment by janpipek** [comment]
Current looks:

image

**Comment by janpipek** [comment]
The last commit allows for links (currently to Wikipedia articles about the cities) but it is not very good UX.

I am looking for alternatives

File: src/charts/world_map.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+import numpy as np
+import pandas as pd
+from bokeh.io import curdoc
+from bokeh.models import HoverTool, OpenURL, TapTool
+from bokeh.plotting import figure, Figure, show
+from bokeh.tile_providers import Vendors, get_provider
+
+# For typing hints
+NumberLike = Union[Real, pd.Series, np.ndarray]
```
Comment (hnykda) [inline src/charts/world_map.py]: :+1: for type hints

**Comment by hnykda** [comment]
image
This is how it looks like on my side, which is great (and link work)! I believe it's fairly close to what we have (and the code is nice and clean).

Do you think you would try to find out how to make it "clickable" through to the first plot, it would be great. But as I said, I am not sure how and if it's possible

**Comment by janpipek** [comment]
I think I got as far as I could with the dots and hovers, though I am not able to change the cursor to hand over the point anyhow :-/

Regarding the links, let's investigate how it would work on the receiving side... Got no idea yet.

File: src/charts/world_map.py (hunk 1 above)
Comment (janpipek) [reply src/charts/world_map.py]: My pleasure.

**Comment by janpipek** [comment]
Last copy looks like this: 

image

Feel free to merge, this is a working POC.

The click through will work once the SERVER_ROOT env.var. points to the right server (or locally by default)

**Comment by hnykda** [comment]
Works, merging.

**Pull request merged**